src/server/Cypher.js

```diff
@@ -62,10 +62,13 @@ router.post("/", async (req, res) => {
     columnNames.forEach((name, i) => {
       dataTypes[name] = columnTypes[i];
     });
-    const currentSchema = await database.getSchema();
-    const isSchemaChanged =
-      JSON.stringify(schema) !== JSON.stringify(currentSchema);
-    schema = currentSchema;
+    let currentSchema = schema;
+    let isSchemaChanged = false;
+    if (mode === "READ_WRITE") {
```

Comment:
There are multiple places in the API codebase now that uses `READ_WRITE`, `READ_ONLY`, and `DEMO`. Try to refactor them into a enum.